deny.toml

```diff
@@ -233,6 +233,8 @@ ignore = [
     "RUSTSEC-2024-0370",
     # lexical-core: Multiple soundness issues, depends on arrow upgrading
     "RUSTSEC-2023-0086",
+    # Use standard library's IsTerminal trait instead of `atty` (unmtainained)
```

Comment:
```suggestion
    # Use standard library's IsTerminal trait instead of `atty` (unmaintained)
```

Reply:
(I assume we need the ignore because a number of our dependencies still use `atty`?)

Reply:
Exactly